src/skin/tooltips.cpp

```diff
@@ -256,12 +270,17 @@ void Tooltips::addStandardTooltips() {
 
     add("maximize_library")
             << tr("Maximize Library")
-            << tr("Maximize the track library to take up all the available screen space.");
+            << tr("Hide all skin sections except the decks to have more")
+            << tr("screen space for the track library.");
```

Comment:
Will these show up as seperate translations?

If so, you can also do this:
```suggestion
            << tr("Hide all skin sections except the decks to have more "
                  "screen space for the track library.");
```

Reply:
oh, I see they can easily be on one line.